pyproject.toml

```diff
@@ -22,8 +22,8 @@ requires = [
 
     # numpy 1.19 was the first minor release to provide aarch64 wheels, but
     # wheels require fixes contained in numpy 1.19.2
-    "numpy==1.19.2; python_version=='3.7' and platform_machine=='aarch64'",
-    "numpy==1.19.2; python_version=='3.8' and platform_machine=='aarch64'",
+    "numpy==1.19.2; python_version=='3.7' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'",
+    "numpy==1.19.2; python_version=='3.8' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'",
```

Comment:
This looks right to me if I'm understanding the situation correctly. I'll probably let Ralf confirm, because this stuff can be deceptively tricky.

One more question I have--is this serious enough to warrant backport to both `1.7.x` series and `1.8.x`. Certainly the latter seems sensible as we're on the precipice of a new release series.

Reply:
Ah, you've targeted this directly at the maintenance branch, that's usually helpful to mention in the PR description because it is easy to miss.

Reply:
Thanks for super quick reply
- this is the patch for 1.7.x, it will break scipy 1.7.x install with pypy3(aarch64), it conflict with
```
    # First PyPy versions for which there are numpy wheels
    "numpy==1.20.0; python_version=='3.7' and platform_python_implementation=='PyPy'",
```
- but for master branch, we already remove above conflict, it's not break, just force user to install v1.19.2 (but actually, no wheel supported), so it's not a break problem https://github.com/scipy/scipy/pull/15318.

Reply:
> Ah, you've targeted this directly at the maintenance branch, that's usually helpful to mention in the PR description because it is easy to miss.

Thanks for reminder, updated